tools/scripts/seed/seedAuthUser.js

```diff
@@ -93,6 +93,63 @@ MongoClient.connect(MONGOHQ_URL, { useNewUrlParser: true }, function(
         emailAuthLinkTTL: null,
         emailVerifyTTL: null
       });
+
+      user.insertOne({
```

Comment:
I think adding an extra user is fine (@raisedadead any objections?), but it needs to be added to the records that will seed deletes.  Same as for `developmentuser`.

Reply:
So... what am I supposed to do?

Reply:
That is correct, we can add extra users.

Reply:
Thanks Mrugesh. 

So, Twaha, instead of 
```js
  user.deleteOne({ _id: ObjectId('5bd30e0f1caf6ac3ddddddb5') }, err => {
```

it would need to delete the new user as well.  `deleteMany` should work here.

Reply:
Done.